scripts/openshiftci-presubmit-unittests.sh

```diff
@@ -7,7 +7,6 @@ set -e
 set -x
 
 export ARTIFACTS_DIR="/tmp/artifacts"
```

Comment:
Remove this this line too. `ARTIFACTS_DIR` is a CI defined variable. Do not overwrite the default artifacts directory  unless you are very sure.